src/main/java/com/contentful/java/cda/CDAContentTypeNotFoundException.java

```diff
@@ -0,0 +1,34 @@
+package com.contentful.java.cda;
+
+import static java.lang.String.format;
+import static java.util.Locale.getDefault;
+
+/**
+ * RuntimeException indicating an entry linking to a non existing Content Type.
+ * <p>
```

Comment:
Yes, [Oracle specified it this way](http://www.oracle.com/technetwork/articles/java/index-137868.html)...